default/static/js/uploader.js

```diff
@@ -171,6 +172,32 @@ var app = {};
         ;
     }
 
+    function download() {
+        window.location = $(this).attr('href');
+    }
+
+    function saveDrive() {
+        showProcessingModal();
+        $.ajax($('.response-success .d-drive').attr('href'), { 'dataType': 'json' })
+            .done(function(data) {
+                $('.google-drive-success .file-google-name').html(data.name);
+                $('.google-drive-success .file-google-id').html(data.id);
```

Comment:
Me imagino que este ID es una cadena de texto ilegible, ¿no? Si es así, no tiene sentido mostrarlo en texto claro. Podrías colocar algo como "Open file" en el span y eliminar esta línea.

Reply:
Es el ID del archivo que se guardó en Drive, lo puse para identificarlo ademas del nombre.

![imagen](https://user-images.githubusercontent.com/48607824/80395045-b0b40280-8880-11ea-9f9e-2eef6a9ab78b.png)

En esta imagen se muestra el mensaje al guardarlo, saco lo que esta entre paréntesis y dejo el otro mensaje?

Aquí un ejemplo cuando no se le da los permisos a Toucan para gestionar la cuenta:

![imagen](https://user-images.githubusercontent.com/48607824/80395284-f8d32500-8880-11ea-8e1d-3d265c391f9d.png)

Reply:
Sacá nomás lo que está entre paréntesis. También deberías ocultar el enlace de "Save to Google Drive" una vez que el archivo se haya guardado.

Por cierto, ¿esos enlaces no eran botones? Podías usar bootstrap para estilarlos como botones aunque sean etiquetas `<a>` por debajo. Entonces, cuando el archivo se guarde en Google Drive correctamente podrías darle un estilo de deshabilitado y sacar el `href` para asegurar de que el archivo no se vuelva a subir.

Al mismo segundo mensaje podés ponerle un enlace (podrías crear un enlace con el texto result.zip y el id que recibiste) para abrir el archivo guardado en una pestaña nueva.

Reply:
"Download" y "Save to Google Drive" son botones ambos, visualmente se ven así por la simplicidad, estética y para que sean iguales nomas, con css los modifique asi.

HTML:
```
<div class="response-success alert alert-info hidden">
    <b>{% trans "Success!" %}</b>
    result.zip(<span class="f-file-size"></span>)
    <a class="btn btn-primary btn-download f-file" href="#" role="button">
        {% trans "Download" %}
    </a>
    <span>  /  </span>
    <button type="button" class="btn btn-primary btn-download d-drive">
        <span>{% trans "Save to Google Drive" %}</span>
    </button>
</div>
```

CSS:
```
.btn.btn-primary.btn-download {
    line-height: 0;
    color: #9baf00;
    background-color: transparent;
    border-color: transparent;
    padding: 0px 0px;
}
.btn.btn-primary.btn-download:hover {
    text-decoration-line: underline;
    color: #6D812B;
}
```

Reply:
"Save to Google Drive" ahora se deshabilita cuando ya se guardo el archivo en el drive. 

En el segundo mensaje, el texto "result.zip" es un botón con enlace que abre el archivo en una pestaña nueva.